mmv1/products/artifactregistry/go_VPCSCConfig.yaml

```diff
@@ -14,6 +14,7 @@
 # Warning: This is a temporary file, and should not be edited directly
 ---
 name: 'VPCSCConfig'
```

Comment:
Could we change this name to VpcscConfig instead? I think it might be equivalent in terms of the output?

Reply:
It changes all of the function names used by the resource, which could hypothetically be used by other resources. I do think it is most likely safe to change, but I would advocate for just using the override for now, and then handling this later (there may be others we want to do the same for).